src/index.ts

```diff
@@ -1,88 +1,71 @@
-import { getDependencies } from './getdependencies';
-import { getRepoIds } from './getrepos';
+import { getDependencies } from './getDependencies';
+import { getRepoIds } from './getRepos';
 const JupiterOneClient = require('@jupiterone/jupiterone-client-nodejs');
-const gql = require("graphql-tag");
+const gql = require('graphql-tag');
 
-const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
-const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
-const accessToken = '';
+require('dotenv').config();
 
 let success = 0, failure = 0;
 const missingDeps = [];
 
-;(async () => {
+(async () => {
   const j1Client = await new JupiterOneClient({
     account: 'j1dev',
     username: 'j1dev',
     password: 'dev',
-    poolId: J1_USER_POOL_ID_PROD,
-    clientId: J1_CLIENT_ID_PROD,
-    accessToken: accessToken,
-    dev: true
+    poolId: process.env.J1_USER_POOL_ID,
+    clientId: process.env.J1_CLIENT_ID,
+    accessToken: process.env.J1_API_TOKEN,
+    dev: true,
   }).init();
+
   const repoMap = await getRepoIds();
-  
-  //value = Repo Name, key = Repo Entity ID
-  for (let value of Array.from(repoMap.values())) {
-    console.log('Repo: ' + value);
 
-    //Get 'From' Repo
-    const mainRepo = await j1Client.queryV1(`FIND CodeRepo WITH name='${value}'`);
-    const depsList = getDependencies(value);
+  for (let repoName of Array.from(repoMap.values())) {
+    console.log('Repo: ' + repoName);
+
+    const mainRepo = await j1Client.queryV1(
+      `FIND CodeRepo WITH name='${repoName}'`,
+    );
+    const depsList = getDependencies(repoName);
+    if (depsList === undefined) {
```

Comment:
`!depsList` is a bit safer and more concise.

Reply:
I have it like this to account for repos that don't have a package.json